server/embed/config.go

```diff
@@ -377,6 +377,11 @@ type Config struct {
 	// The gateway translates a RESTful HTTP API into gRPC.
 	EnableGRPCGateway bool `json:"enable-grpc-gateway"`
 
+	// ExperimentalEnableGRPCDump enables saving requests & responses to dump files
```

Comment:
Can we make this more generic? e.g., `s/EnableGRPCDump/EnableRequestDump/`?

Reply:
I think using GRPC is a little bit misleading here. I changed it to KVRequest. This is a better description to this dump & replay feature for now.